pyproject.toml

```diff
@@ -15,6 +15,7 @@ python = "^3.7"
 Flask = {extras = ["async"], version = "^2.0.2"}
 flask-cors = "^3.0.10"
 cairo-lang = "0.7.1"
+py-solc-x = "^1.1.1"
```

Comment:
Can it be a dev dependency?

Reply:
Or is it even needed? Check this comment: https://github.com/Shard-Labs/starknet-devnet/pull/41#discussion_r814783789

Reply:
Removed according to the comment